tests/root_io/CMakeLists.txt

```diff
@@ -47,8 +47,10 @@ set_property(TEST check_benchmark_outputs PROPERTY DEPENDS read_timed write_time
 if (DEFINED CACHE{PODIO_TEST_INPUT_DATA_DIR})
   message(STATUS "Using test inputs stored in: "  ${PODIO_TEST_INPUT_DATA_DIR})
   add_executable(read-legacy-files-root read-legacy-files-root.cpp)
-  target_link_libraries(read-legacy-files-root PRIVATE TestDataModel TestDataModelDict podio::podioRootIO)
-
+  target_link_libraries(read-legacy-files-root PRIVATE TestDataModel podio::podioRootIO)
+  if (ENABLE_SIO)
+    target_link_libraries(read-legacy-files-root PRIVATE TestDataModelSioBlocks)
```

Comment:
Similar to above, this should not be necessary as this is loaded dynamically at runtime.